lib/vagrant-ovirt3/action.rb

```diff
@@ -13,7 +13,9 @@ def self.action_up
           b.use ConnectOVirt
           b.use Call, IsCreated do |env, b2|
             if env[:result]
-              b2.use MessageAlreadyCreated
+              b2.use StartVM
```

Comment:
lines 16-18 will attempt to bring it up when it is already up. is that an intended change?
Lines 31-33 take care of this.